Started grammar and puncuation editing

KelvinShadewing opened the pull request:
Started going through files and editing the contents for grammar, punctuation, and spelling. More fixes to come later; it's bed time for me now.

fluteds commented:
Is this done with American English rules and spelling? I'm happy to help out if you need some extra eyes!

Holsterbau commented on src/aelplermagronen.md:
```diff
@@ -19,19 +19,20 @@ Cooking time: ~30 minutes
 ## Contributors
 
 - **Alexander Bocken** -- [contact](mailto:alexander@bocken.org)
+- Kelvin Shadewing - https://odysee.com/@KelvinShadewing:d
```
Should be `[Channel](https://odysee.com/@KelvinShadewing:d)`

KelvinShadewing commented:
Removed contribution links from minor edits. Figured it'd get obnoxious crediting myself just for a few missing periods and commas everywhere. Also, I fixed the issues the system mentioned, but it's still saying my tofu stir fry recipe is failing checks, just won't say why now.

KelvinShadewing commented:
Does anyone know why my recipe keeps getting rejected? I've compared it to several others and can't find the difference that's making the compliance check fail.

Socerest2 commented:
> Does anyone know why my recipe keeps getting rejected? I've compared it to several others and can't find the difference that's making the compliance check fail.

Yeah It looks like to much spacing, I'll fix em up.

Socerest2 commented on src/arroz-chaufa.md:
```diff
@@ -23,13 +23,13 @@ Peruvian-chinese dish, Easy to cook just add and mix everything.
 
```
```suggestion
```
Fix Spacing / Why Check Is Failing

KelvinShadewing commented:
Finally got it! Heck, this thing sure is strict about spacing. Also sucks I couldn't have "Simple Cooking Procedures" instead of "Directions". Oh well. XD

Socerest2 commented on src/assam-tea.md:
```diff
@@ -20,7 +20,7 @@ This is a simple Assam tea recipe.
-4. Add sugar and milk to taste
+4. Add sugar and milk to taste.
 
 
```
```suggestion

```
Fixing Spacing / Why Check Is Failing

Socerest2 reviewed (changes requested):
Spacing fix

Socerest2 reviewed (approved):
Look good now!

fluteds commented on src/aelplermagronen.md:
```diff
@@ -19,19 +19,20 @@ Cooking time: ~30 minutes
 7. Add macaroni and the remaining milk to cover everything. Most of the milk will be absorbed by the macaroni.
 8. Shred your cheese.
-9. A minute or two before the macaroni are done, add the shredded cheese into the pot. It should appear a bit too runny in the pot. While cooling it will increase in viscosity quite a bit. If the final texture is not creamy enough it is most likely due to using the wrong cheese.
+9. A minute or two before the macaroni are done, add the shredded cheese into the pot. It should appear a bit too runny in the pot. While cooling it will increase in viscosity quite a bit. If the final texture is not creamy enough, it is most likely due to using the wrong cheese.
```
I would suggest changing the `the macaroni are done` to `the macaroni is done` as it makes more grammatical sense

KelvinShadewing replied:
IIRC, macaroni is also plural, but it does sound better using 'is' instead of 'are'. Thanks.
